engine/gamesys/src/gamesys/scripts/script_image.cpp

```diff
@@ -12,23 +12,25 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#include <assert.h>
```

Comment:
Unused?
And if it _is_ used, we're in scripting domain, where lua_error would be preferred?